### ISSUE TITLE
[BUG] Products not found to overlap, although they should

### ISSUE THREAD

**Issue by rzinke** [aria-tools/ARIA-tools#309]
**Describe the bug**
I am running ```ariaTSsetup.py``` on a large number of frames. The frames appear to be spatially continuous based on the ```ariaPlot.py``` results. When I run the TS setup routine, however, I get the error message: ***ERROR: Products do not overlap or were not provided in a contigious sorted list.***

**To Reproduce**
1. Download
```
ariaDownload.py -t 165 -b "23.0 55.0 63.0 117.0" -s 20141001 -e 20151001
```

2. Command used
```
ariaTSsetup.py -f "products/*.nc" -l all -d download -m download -nt 8 -mo 390000 -ml 3 -b /data/Tibet/Track165/AOIs/track_165_TibetA/track_165_TibetA.shp
```

3. Error Output
```
NETCDF:"/data/Tibet/Track165/AllProducts/S1-GUNW-D-R-165-tops-20150930_20141029-003515-00080E_00030N-PP-033b-v2_0_5.nc":/science/grids/data/connectedComponents is GDAL compatible
ERROR: Products do not overlap or were not provided in a contigious sorted list.
Traceback (most recent call last):
  File "/home/rzinke/miniconda3/envs/insar/bin/ariaTSsetup.py", line 18, in <module>
    main(inps)
  File "/home/rzinke/miniconda3/envs/insar/lib/python3.10/site-packages/ARIAtools/tsSetup.py", line 458, in main
    export_products(standardproduct_info.products[1],
  File "/home/rzinke/miniconda3/envs/insar/lib/python3.10/site-packages/ARIAtools/extractProduct.py", line 669, in export_products
    product_stitch_overlap(unw_files,conn_files,prod_bbox_files,bounds,prods_TOTbbox, outFileUnw=outFileUnw,outFileConnComp= outFileConnComp, mask=mask,outputFormat = outputFormat,verbose=verbose)
  File "/home/rzinke/miniconda3/envs/insar/lib/python3.10/site-packages/ARIAtools/unwrapStitching.py", line 1458, in product_stitch_overlap
    unw.UnwrapOverlap()
  File "/home/rzinke/miniconda3/envs/insar/lib/python3.10/site-packages/ARIAtools/unwrapStitching.py", line 340, in UnwrapOverlap
    self.__calculateCyclesOverlap__()
  File "/home/rzinke/miniconda3/envs/insar/lib/python3.10/site-packages/ARIAtools/unwrapStitching.py", line 369, in __calculateCyclesOverlap__
    raise Exception
Exception
```

**Expected behavior**
Given that the frames look continuous in ariaPlot.py -plottrack, I would expect them to be continuous across the AOI.

**Desktop (please complete the following information):**
 - ARIA-tools git tag: 092d052e5c332246454162f6f6f5cfcf7b8f3b97
 - OS: Ubuntu 20.04.4 LTS (Focal Fossa)

**Comment by rzinke** [comment]
[Shapefile.zip](https://github.com/aria-tools/ARIA-tools/files/9006286/Shapefile.zip)
Here is the shapefile I used.

**Comment by bbuzzanga** [comment]
Fixed with #313
